shamu/Android.mk

```diff
@@ -19,203 +19,4 @@ LOCAL_PATH := $(call my-dir)
 
 ifeq ($(TARGET_DEVICE),shamu)
 
-include $(CLEAR_VARS)
-LOCAL_MODULE := libtime_genoff
-LOCAL_MODULE_OWNER := motorola
-LOCAL_SRC_FILES := proprietary/vendor/lib/libtime_genoff.so
-LOCAL_MULTILIB := 32
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_CLASS := SHARED_LIBRARIES
-LOCAL_MODULE_SUFFIX := .so
-LOCAL_VENDOR_MODULE := true
-include $(BUILD_PREBUILT)
```

Comment:
Looks like a horrible idea to me

Reply:
You are aware this is migrated to Android.bp as of this change, yes?

Reply:
> You are aware this is migrated to Android.bp as of this change, yes?

I'm not. Thanks for bringing that up. I really should look at the changes appropriately before commenting on them. Again thank you.